src/telemetry/index.ts

```diff
@@ -40,14 +75,15 @@ export class TelemetryService implements vscode.Disposable {
   }
 
   sendTelemetryError(
-    eventName: string,
+    eventName: string, // TODO: should be TelemetryEvents
```

Comment:
Open TODO

Reply:
This is to be addressed after we migrate all telemetry events to this new format